src/components/common/ui/form/dropdown-mui/Dropdown.styles.tsx

```diff
@@ -0,0 +1,148 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+import { FieldState } from '@/components/common/ui/form/common/types';
+
+import { DropDownSize } from './Dropdown';
+
+const color = {
+  default: 'grey.600',
+  disabled: 'grey.400',
+  error: 'error.500',
+  success: 'success.600',
+};
+
+const inputHeight = {
+  small: 40,
+  medium: 46,
+  large: 52,
+};
+
+export const popper = (
+  numberOfOptions: number,
+  size: number,
+): SxProps<Theme> => ({
+  top: '8px !important',
+  backgroundColor: 'transparent',
+  '& ul': {
+    borderColor: 'grey.400',
+    border: '2px solid',
+    borderRadius: '8px',
+  },
+  '& .MuiPaper-root': {
+    borderRadius: '8px',
+    backgroundColor: 'backgroundDark.100',
+
+    '& .MuiAutocomplete-noOptions': {
+      border: '2px solid',
+      borderColor: 'grey.600',
+      borderRadius: '8px',
+      color: 'grey.400',
+    },
+  },
+  '& .MuiAutocomplete-listbox': {
+    height: `${numberOfOptions * size + 16}px`, //16 is for padding
```

Comment:
remove `size` prop and look for class for separate item and adjust it need height using paddings, font size and line height